core/src/main/AndroidManifest.xml

```diff
@@ -1 +1,3 @@
-<manifest package="com.klaviyo.coresdk" />
\ No newline at end of file
+<manifest package="com.klaviyo.coresdk" />
+<uses-permission android:name="android.permission.INTERNET" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
i think you'll need to put the uses-permission tags inside of manifest:
`<manifest ..>
    <uses-permission... />
</manifest>`